sampler.go

```diff
@@ -326,8 +326,8 @@ func newAdaptiveSampler(strategies *sampling.PerOperationSamplingStrategies, max
 	samplers := make(map[string]*GuaranteedThroughputProbabilisticSampler)
 	for _, strategy := range strategies.PerOperationStrategies {
 		samplers[strategy.Operation] = newGuaranteedThroughputProbabilisticSampler(
-			strategies.DefaultLowerBoundTracesPerSecond,
 			strategy.ProbabilisticSampling.SamplingRate,
+			strategies.DefaultLowerBoundTracesPerSecond,
 			defaultMaxSamplesPerSecond, // TODO (wjang) get the maxSamplesPerSecond from strategy
```

Comment:
what is the plan for this one? Should we land if before this PR?

Reply:
Now what? #143 landed.